Assets/Modules/Hero/Scripts/ControlManager.cs

```diff
@@ -0,0 +1,127 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Aloha
+{
+    /// <summary>
+    /// Manager that allow to control mouse hands
+    /// </summary>
+    public class ControlManager : MonoBehaviour
+    {
+        [SerializeField]
+        protected GameObject rightHand;
+        [SerializeField]
+        protected GameObject leftHand;
+
+        // Update is called once per frame
```

Comment:
commentaire non conforme :
```suggestion
/// <summary>
/// Update is called once per frame
/// </summary>
```